KimeraRPGO/GenericSolver.h

```diff
@@ -60,6 +60,6 @@ class GenericSolver {
   bool debug_;
 };
 
-} // namespace KimeraRPGO
+}  // namespace KimeraRPGO
 
-#endif
\ No newline at end of file
+#endif  // KIMERARPGO_GENERICSOLVER_H_
```

Comment:
Much nicer! Btw I would use pragma once or the standard is to write the filespath to ensure it is unique